src/errors/InvalidAccessError.ts

```diff
@@ -0,0 +1,5 @@
+export class InvalidAccessError extends Error {
+  constructor() {
+    super('Cannot directly change type. Elements will naturally inherit their Parent type.');
```

Comment:
Here I think we'd want to say "StructureDefinitions will naturally..." instead of "Elements will naturally...", since StructureDefinition is the FHIR terminology for what we are exporting.